Dockerfiles/agent-ot/Dockerfile.agent-otel

```
@@ -26,7 +26,7 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/*
 
 # TEMP: Use github source code
-RUN git clone --filter=blob:none --branch "${AGENT_BRANCH}" --single-branch https://github.com/DataDog/datadog-agent.git datadog-agent-${AGENT_VERSION}
+RUN git clone --filter=blob:none --branch "${AGENT_BRANCH}" --single-branch --depth 1 https://github.com/DataDog/datadog-agent.git datadog-agent-${AGENT_VERSION}
```

```suggestion
RUN git clone --filter=blob:none --branch "${AGENT_BRANCH}" --single-branch https://github.com/DataDog/datadog-agent.git datadog-agent-${AGENT_VERSION}
```
this builds fine locally, I bet it would pass the pipeline too